torch_glow/tests/utils.py

```diff
@@ -23,17 +27,28 @@ def jitVsGlow_(f_torch, f_glow, *inputs):
 
         # check that there are no Glow nodes in the torch graph
         torch_graph = torch_trace.graph_for(*inputs)
-        print("torch_graph,", torch_graph)
         num_glow_nodes = len(torch_graph.findAllNodes(GLOW_NODE_NAME))
         assert num_glow_nodes == 0, "Expected no Glow nodes, found {}".format(
             num_glow_nodes
         )
 
-        # check that there is exactly 1 Glow node in the glow graph
+        # Check that only ops in expected_fused_ops were fused
         glow_graph = glow_trace.graph_for(*inputs)
-        print("glow_graph,", glow_graph)
-        num_glow_nodes = len(glow_graph.findAllNodes(GLOW_NODE_NAME))
-        assert num_glow_nodes == 1, "Expected exactly 1 Glow node, found {}".format(
-            num_glow_nodes)
+        for node in glow_graph.nodes():
+            kind = node.kind()
+            if kind == GLOW_NODE_NAME:
+                # If the node is a Glow compilation group, check that all ops inside it are in expected_fused_ops
+                glow_group = node.g(SUBGRAPH_ATTR)
+
+                for op in glow_group.nodes():
+                    op_kind = op.kind()
+                    if "prim::" not in op_kind:
+                        assert op_kind in expected_fused_ops, "Did not expect {} to be fused".format(
+                            op_kind)
+            else:
+                # If the node is not a Glow compilation group, check that it is *not* in expected_fused_ops
+                if "prim::" not in kind:
```

Comment:
is this prim:: check needed?

Reply:
I noticed that some constants in the IR labelled as `Prim::Constants` get moved into the Glow compilation group and some don't based on what is and is not in the group. Maybe I should make this more specific?

Reply:
True, sounds good let's keep it